org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java

```diff
@@ -91,12 +72,51 @@ public static String getNodeJsLocation() {
 		return null;
 	}
 
+	private static void validateNodeVersion() {
+
+		String nodeVersion = null;
+		String[] nodeVersionCommand = new String[] { "/bin/bash", "-c", "node -v" };
+		if (Platform.getOS().equals(Platform.OS_WIN32)) {
+			nodeVersionCommand = new String[] { "cmd", "/c", "node -v" };
+		}
+
+		try (BufferedReader reader = new BufferedReader(
+				new InputStreamReader(Runtime.getRuntime().exec(nodeVersionCommand).getInputStream()));) {
+			nodeVersion = reader.readLine();
+		} catch (IOException e) {
+			Activator.getDefault().getLog().log(
+					new Status(IStatus.ERROR, Activator.getDefault().getBundle().getSymbolicName(), e.getMessage(), e));
+		}
+
+		final String version = nodeVersion.startsWith("v") ? nodeVersion.replace("v", "") : nodeVersion;
+
+		boolean isSupportedVersion = supportedNodejsVersions.stream().anyMatch(new Predicate<String>() {
```

Comment:
It could just become something like `SUPPORTED_NODEJS_MAJOR_VERSIONS.contains(version.getMajor())`.